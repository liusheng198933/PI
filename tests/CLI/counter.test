#!/bin/sh
./run_one_test.py $srcdir/testdata counter stats.json
