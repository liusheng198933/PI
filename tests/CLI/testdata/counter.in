counter_reset CounterA 9
counter_read CounterA 9
counter_write CounterA 9 packets=7 bytes=9993
counter_read CounterA 9
counter_write CounterA 9 packets=12
counter_read CounterA 9
counter_reset CounterA 9
counter_read CounterA 9
