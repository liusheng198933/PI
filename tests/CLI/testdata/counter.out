????
Device assigned successfully.
Selecting device.
????
????
Dumping counter entry:
	Packets: 0
	Bytes: 0
????
Dumping counter entry:
	Packets: 7
	Bytes: 9993
????
Dumping counter entry:
	Packets: 7
	Bytes: 9993
????
Dumping counter entry:
	Packets: 12
????
Dumping counter entry:
	Packets: 12
	Bytes: 9993
????
????
Dumping counter entry:
	Packets: 0
	Bytes: 0
????
